src/agent/protocols/hack/update-generated-proto.sh

```diff
@@ -1,68 +1,138 @@
 #!/bin/bash
 
+# //
+# // Copyright 2020 Ant Financial
+# //
+# // SPDX-License-Identifier: Apache-2.0
+# //
+
 die() {
-    echo $1
-    exit
+    echo "====================================================================" 1>&2
```

Comment:
Just an idea, but you could simplify this further to something like:

```sh
cat <<EOT >&2
====================================================================

$1

====================================================================
EOT
```

Reply:
Yes, of course.  😄